edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_extras.scss

```diff
@@ -1,7 +1,358 @@
+  background-color: $primary;
+  border: 1px solid $primary;
+  border-color: $primary;
+}
+
+.wrapper-prompt div.prompt.warning div.copy {
+  background: $gray-l5;
+  border-top-color: $primary;
+}
+
+.contact-us-wrapper .btn-primary,
+.contact-us-wrapper .btn-brand {
+  border-color: $primary;
+}
+
+.contact-us-wrapper .btn-primary:hover,
+.contact-us-wrapper .btn-brand:hover,
+.contact-us-wrapper .btn-primary:focus,
+.contact-us-wrapper .btn-brand:focus {
+    background-image: none;
+    background-color: $primary;
+    border-color: $primary;
+}
+
+@media (min-width: 768px) {
+  .content-wrapper {
+     padding: 0 0 10px;
+  }
+}
+
+.submit-attempt-container {
+  .btn-primary:disabled, .btn-brand:disabled,
+  .btn-primary.is-disabled, .is-disabled.btn-brand {
+    border-color: $pearson-grey;
+    background: $pearson-grey;
+  }
+}
+
+input.js-post-title.field-input,
+form.forum-new-post-form input[type=text].field-input,
+form.forum-new-post-form input[type=text].wrapper-custom-select,
+.edit-post-form input[type=text].field-input,
+.edit-post-form input[type=text].wrapper-custom-select {
+  color: $primary-font-color;
+}
+
+div.wmd-panel.wmd-preview p {
+  color: $inverse;
+}
+
+
+
+//static-tamplate
+
+section.container.about {
+  border-left: 1px solid $primary;
+  margin-top: 50px;
+  margin-bottom: 30px;
+
+  h1,
+  h2,
+  a, {
+    color: $primary;
+  }
+
+  ol {
+    list-style-type: none;
+  }
+}
+
+// Rocket chat tab service
+
+iframe.rocket-chat-view {
+  width: 100%;
+  height: 500px;
+  border: none;
+}
+
+.rocket_chat_error_message {
+  text-align: center;
+  color: red;
+}
+
+.alert.warning .action-secondary,
+.notification.warning .action-secondary,
+.prompt.warning .action-secondary {
+  color: $primary;
+}
+
+.course-info .pay-now-button.btn {
+  background-color: $pearson-dark-blue-courseware;
+  border: 1px solid $pearson-dark-blue-courseware;
+  padding: 10px 0;
+  width: 40%;
+  display: block;
+  font: normal 1.2rem/1.6rem Open Sans;
+  text-align: center;
+}
+
+.modal .inner-wrapper p {
+  font-size: 0.9em;
+}
+
+.modal .inner-wrapper .close-modal:focus {
+  outline: none;
+}
+
+.u-field-date_joined .u-field-title {
+  cursor: text;
+}
+
+.view-profile .tabpanel {
+  .badge-set-display {
+    min-width: auto;
+
+    .badge-list {
+      padding: 0;
+    }
+
+    .badge-display {
+      width: 100%;
+    }
+  }
+}
+
+@media (min-width: 768px) {
+  .view-profile .tabpanel {
+    .badge-set-display {
+      min-width: auto;
+    }
+  }
+}
+
+.signin-logo {
+  display: block;
+  width: 400px;
+  right: 110px;
+  position: absolute;
+  top: 100px;
+}
+
+.signin-container {
+  background: white;
+  padding: 30px;
+  border-radius: 10%;
+  box-shadow: 3px 2px 2px 5px #bcb9b9;
+  position: absolute;
+  right: 50px;
+  top: 250px
+}
+
+.signing-title {
+  text-align: center;
+  font-size: 18px;
+  font-weight: 600;
+  margin-bottom: 10px;
+}
+
+.login-register .form-wrapper .login-providers {
```

Comment:
Can we get the colour values from a variable? @luismorenolopera

Reply:
done